docs/change_resource_hierarchy.md

```diff
@@ -111,11 +127,12 @@ Review the `tf-wrapper.sh`. It is a bash script helper responsible for applying
         echo "$(echo ${component} | sed -r 's/-/\//g' )/${env} doesn't match ${branch}; skipping"
       fi
 
-      component=$old_component #🟢 Assign old component value before next while-loop iteration
+      #🟢 Assign old component value before next while-loop iteration
+      component=$old_component
     done
     ```
 
-1. Create a new function `check_env_path_folder`.
+1. Create the new function `check_env_path_folder`.
```

Comment:
Just wondering, in order to allow the user to understand the logic in this modifications, shouldn`t this function be created before its calling in the old functions?

Reply:
I think it is a good idea.
I will move the function.